docs/docs/cli/login/README.md

```diff
@@ -23,6 +23,14 @@ Once you're done, go back to your shell and you should see confirmation that you
 Logged in as dylburger (dylan@pipedream.com)
 ```
 
+::: warning
+
+If you are a member of a [workspace](/docs/workspaces/) then you'll also need to configure the `pd` CLI with your workspace ID.
```

Comment:
I don't know if all users will know what this means, but since it's now the default, I would remove the warning section and just include it as a part of the standard instructions for logging into the CLI. Since we'll be migrating existing users to workspaces soon, this will also apply to them.

Reply:
Good point, moved it as a general instruction as part of logging in with the CLI.

Best case scenario, we add workspace configuration generation as part of `pd login` and `pd signup`.